common/config/azure-pipelines/jobs/regression-testing.yaml

```diff
@@ -17,27 +17,93 @@ schedules:
         - release/*
 
 jobs:
-  - template: ci-core.yaml
-    parameters:
-      name: Node_16
-      nodeVersion: 16.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
-    parameters:
-      name: Node_14_x
-      nodeVersion: 14.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
-    parameters:
-      name: Node_12
-      nodeVersion: 12.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
+- job:
+  variables:
+  - name: linuxPool
```

Comment:
were these pool changes necessary in this PR or just for clarity?

Reply:
I did this to keep all similar files basically the same, and because with core-build being the new template, I did not give it have the same platform variable anymore.